appeals/web/src/server/views/patterns/display-page.pattern.njk

```diff
@@ -48,7 +48,7 @@
 {%- block pageContent -%}
 	{%- set formComponents = [] -%}
 	{%- for component in pageContent.pageComponents -%}
-		{%- if component.type === 'radios' or component.type === 'checkboxes' or (component.type === 'inset-text' and formComponents | length) -%}
+		{%- if component.type === 'radios' or component.type === 'checkboxes' or (component.type === 'inset-text' and formComponents | length) or (component.type === 'warning-text' and formComponents | length) -%}
```

Comment:
I'd be worried about this breaking other pages that use this pattern template, but it seems ok judging by the snapshots. Just checking, does the warning text definitely need to be within the form, or could you achieve the same visual result without this check?

Reply:
no I did go through with Alex - not ideal I know, but that custom way of extracting the form fields doesn't help. If I put it outside the form won't be able to position where required...